include/amici/rdata.h

```diff
@@ -101,12 +101,16 @@ class ReturnData {
      * nplist x ny, row-major) */
     std::vector<realtype> ssigmay;
     
-    /** observable (dimension: nt x ny, row-major) */
+    /** observable (dimension: nt*ny) */
     std::vector<realtype> res;
 
-    /** parameter derivative of residual (dimension: nt x ny x nplist,
+    /** parameter derivative of residual (dimension: nt*ny x nplist,
```

Comment:
Why * instead x?

Reply:
its 1D ...

Reply:
because its actually a 2D matrix.

Reply:
See above. Maybe I don't get it, but for me it doesn't seem to matter whether one calls this 1D, flattened 2D, or flattened 3D. The order of the elements should still be defined somehow.